[2023-12-10T17:51:32.441+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test.merge_df manual__2023-12-10T12:51:10.603168+00:00 [queued]>
[2023-12-10T17:51:32.450+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test.merge_df manual__2023-12-10T12:51:10.603168+00:00 [queued]>
[2023-12-10T17:51:32.450+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-10T17:51:32.481+0500] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): merge_df> on 2023-12-10 12:51:10.603168+00:00
[2023-12-10T17:51:32.491+0500] {standard_task_runner.py:57} INFO - Started process 7380 to run task
[2023-12-10T17:51:32.503+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'merge_df', 'manual__2023-12-10T12:51:10.603168+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmptywgkcbs']
[2023-12-10T17:51:32.509+0500] {standard_task_runner.py:85} INFO - Job 28: Subtask merge_df
[2023-12-10T17:51:32.600+0500] {task_command.py:416} INFO - Running <TaskInstance: test.merge_df manual__2023-12-10T12:51:10.603168+00:00 [running]> on host xypma-VirtualBox
[2023-12-10T17:51:32.708+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='test' AIRFLOW_CTX_TASK_ID='merge_df' AIRFLOW_CTX_EXECUTION_DATE='2023-12-10T12:51:10.603168+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-10T12:51:10.603168+00:00'
[2023-12-10T17:51:32.710+0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-10T17:51:32.710+0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/xypma/lab3/MLOps_lab3/ && python /home/xypma/lab3/MLOps_lab3/scripts/merge_df.py']
[2023-12-10T17:51:32.720+0500] {subprocess.py:86} INFO - Output:
[2023-12-10T17:51:34.810+0500] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-10T17:51:34.841+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test, task_id=merge_df, execution_date=20231210T125110, start_date=20231210T125132, end_date=20231210T125134
[2023-12-10T17:51:34.873+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-10T17:51:34.898+0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
